src/main/webapp/app/overview/header-course.component.ts

```diff
@@ -63,4 +63,8 @@ export class HeaderCourseComponent implements OnChanges {
     redirectToCourseManagement() {
         this.router.navigate(['course-management', this.course.id]);
     }
+
+    shouldShowGoToStudentViewButton() {
```

Comment:
Please move this directly into the template instead of using a method (methods are called repeatedly and should be avoided in the template if possible)